catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/miro/MiroWorkType.scala

```diff
@@ -0,0 +1,17 @@
+package uk.ac.wellcome.platform.transformer.transformers.miro
+
+import uk.ac.wellcome.models.work.internal.WorkType
+
+trait MiroWorkType {
+
+  /** We set the same work type on all Miro images.
+    *
+    * This is based on the Sierra work types -- we'll want to revisit this
+    * when we sort out work types properly, but it'll do for now.
+    */
+  def getWorkType: Option[WorkType] =
+    Some(WorkType(
+      id = "q",
+      label = "Digital Images"
```

Comment:
For consistency, this comes through from Sierra as `Digital images`. Obviously all this changes when we're picking up from our own list, like location types.